scripts/create-project.ts

```diff
@@ -78,7 +78,7 @@ cli.main(async () => {
     }
 
     /** we used to use the project service create method here, but we shouldn't need to */
-    copyFolderRecursiveSync(templateFolderDirectory, projectsRootFolder)
+    cpAsync(templateFolderDirectory, projectsRootFolder)
```

Comment:
this needs to be awaited

Reply:
done... I'm still digging into the parent.remove error making the tests check to fail, following the destroyEngine parth, even though my code isn't touching any of that by searching globally in the project, I found that sometimes it's being awaited and sometimes it isn't, this can be confirmed by using the IDE search tool for destroyEngine